lib/src/arg_parser_exception.dart

```diff
@@ -9,6 +9,6 @@ class ArgParserException extends FormatException {
   /// This will be empty if the error was on the root parser.
   final List<String> commands;
 
-  ArgParserException(super.message, [Iterable<String>? commands])
+  ArgParserException(super.message, super.source, [Iterable<String>? commands])
```

Comment:
This is a public class so this would be a breaking change if it is a required parameter. And the value added here is probably not sufficient to do a breaking change (which would be very painful for this package).

We also use positional optional params here unfortunately so the only non-breaking option would be to add this to the end of that list (after `commands`).